src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -23,18 +23,11 @@ public class Settings : INotifyPropertyChanged
     {
         private enum CmdArgs
         {
-            WorkAreaSize = 1,
-            TargetWorkAreaSize = 2,
-            PowerToysPID,
-        }
-
-        private enum WorkAreaCmdArgElements
-        {
-            X = 0,
-            Y,
-            Width,
-            Height,
-            DPI,
+            MonitorsCount = 0,
+            MonitorName = 1,
+            PowerToysPID = 2,
+            SpanZones = 3,
+            DPI = 4,
```

Comment:
To simplify the arguments parsing, wouldn't be better to put `PowerToysPID` as first argument and `SpanZones` as second?
Then `MonitorsCount` followed by pairs of `MonitorName` and `DPI` for each monitor.